src/features/registry/registry.ts

```diff
@@ -34,10 +34,17 @@ const SUPERVISOR_REPOSITORIES = /^resin\/(?:[a-zA-Z0-9]+-)+supervisor$/;
 
 const NEW_REGISTRY_REGEX = /(^(\d+)\/[\d\-]+$|^(v2\/[a-z0-9]+)(-[0-9]+)?)/;
 
+// match <org>/<app>/<semver|commit>/<service> where <semver|commit> and <service> are optional
```

Comment:
No, we can't use the tag field (after the colon) for redirecting to different registries since each release is a unique registry with unique permissions but the docker client doesn't request the tag or digest until after authentication has taken place.

So whatever is included as a tag is ignored (and isn't available as part of an auth scope anyway).